### PR TITLE
Updated Display to show some additional values

### PR DESCRIPTION
**Pull request by babusid**

### CODE REVIEW

**Review by tiandahuang — DISMISSED** [review]

File: Drivers/Src/MotorController.c [hunk 1]
```diff
@@ -8,13 +8,13 @@
 #define MOTOR_VELOCITY 0x243
```
Comment (ClarkPoon) [inline Drivers/Src/MotorController.c:8]: I'm going to assume all of the changes here are for formatting and not read them

File: Tests/Test_IgnitionSwitches.c [hunk 2]
```diff
@@ -0,0 +1,36 @@
+/*
+ *Test for Ignition Switch
```
Comment (ClarkPoon) [inline Tests/Test_IgnitionSwitches.c:2]: Does this actually test the ignition switch?

**Comment by Cam0Cow** [comment]
Is there any reason to keep this PR open, or can it be closed? @babusid

**Comment by babusid** [comment]
> Is there any reason to keep this PR open, or can it be closed? @babusid 

Think this is probably an @IshDeshpa question, but I don't see any reason off the top of my head

**Comment by IshDeshpa** [comment]
Didn't you force merge it into master? @babusid

**Comment by babusid** [comment]
> Didn't you force merge it into master? Or was that a different branch... @babusid 

Yeah I think so, display does everything we want it to rn. I'll close it but keep the branch open for now.

**Pull request closed without merging**